home/tests/integration/views/api/test_export.py

```diff
@@ -25,7 +25,7 @@ def test_export_users(self):
 
         response = c.get(f"/api/export/users?contest_id={self.contest0_id}")
         self.assertEqual(200, response.status_code)
-        self.assertEqual("application/octet-stream", response["Content-Type"])
+        self.assertEqual("text/csv", response["Content-Type"])
```

Comment:
Looks like the Django upgrade changed the behavior to "default" to `text/csv` probably (which is nice  and definitely what we want, so sweet!!!)

But this probably indicates that the header wasn't explicitly set and relied on a default fallback.
Instead of relying on that can you also explicitly set this header in the controller for this endpoint?